test/README.md

```diff
@@ -0,0 +1,108 @@
+# Media Insights Engine Testing - How To
+
+### Overview
+
+MIE Framework has 3 types of tests:
+
+
+*Unit:* Tests of each functional component of the framework at the code level, e.g. scope is the code itself, checks for introduction of bugs by syntax errors, whitespace issues, correct flow control and order of execution, etc
+
+*These tests can be run locally without an MIE deployment. 
+
+*Integration:* Tests of each functional component of the framework with the dependent cloud resources, e.g. scope is the ensure no bugs introduced by dependencies and that a change did not introduce a feature that is incompatible with dependencies
```

Comment:
Minor: It might help to give a concrete example of each kind of test here to help reinforce the distinction.  Especially integration vs. e2e.  Unit tests stand on their own since they do not require a deployment to run.

Reply:
Good call. I'll write up something quickly.